rabbitmq/docker-healthcheck

```diff
@@ -1,11 +1,14 @@
 #!/bin/bash
 set -eo pipefail
 
-host="$(hostname -s || echo 'localhost')"
-export RABBITMQ_NODENAME="${RABBITMQ_NODENAME:-"rabbit@$host"}"
```

Comment:
Isn't this `export` still necessary to ensure `rabbitmqctl` works/connects properly?

Reply:
`rabbitmqctl` already defaults to that (ctag-style jumps):

* https://github.com/rabbitmq/rabbitmq-server/blob/v3.7.10/scripts/rabbitmqctl#L30
* https://github.com/rabbitmq/rabbitmq-server/blob/v3.7.10/scripts/rabbitmq-env#L237
* https://github.com/rabbitmq/rabbitmq-server/blob/v3.7.10/scripts/rabbitmq-env#L152-L153
* https://github.com/rabbitmq/rabbitmq-server/blob/v3.7.10/scripts/rabbitmqctl#L32
* https://github.com/rabbitmq/rabbitmq-server/blob/v3.7.10/scripts/rabbitmq-env#L408

To approach this from a different perspective, `docker run -it rabbitmq rabbitmqctl help` is expected to just work, as well as `docker exec -it my-rabbitmq-node rabbitmqctl status`. Same applies to healthcheck 😉

Reply:
Awesome, makes sense; thanks for the detailed clarification and references! 😀